src/index.js

```diff
@@ -1,13 +1,21 @@
-import 'asm-dom/cpp/';
-import './assets/index.css';
+import "asm-dom/cpp/"
+import "./assets/index.css"
 
-let loader;
-const config = {};
+let loader
+const config = {}
 
-if ('WebAssembly' in window) {
-  loader = import('../glue/wasm.js');
+if ("WebAssembly" in window) {
+	loader = import("../glue/wasm.js")
 } else {
-  loader = import('../glue/asm.js');
+	loader = import("../glue/asm.js")
 }
 
-loader.then(module => module.default(config));
+loader.then(module => module.default(config))
+
+// Parcel right now doesn't support --reload
+// and its HMR is not working on with asm-node, hence hard-reload
+if (module.hot) {
```

Comment:
Cool, looks good to me. But what if we put this code in the files in the `glue` folder? In this way logic is hidden to the final user. Also, `index.js` might be modified with other imports that doesn't require a complete reload of the page